src/test/java/org/cirdles/topsoil/app/utils/GetDocumentsDirectoryOperationTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * To Change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+ package org.cirdles.topsoil.app.utils;
+
+ import java.nio.file.Path;
+ import org.junit.Test;
+ import static org.junit.Assert.*;
+
+ /**
+  *
+  * @author kyser
+  */
+ public class GetDocumentsDirectoryOperationTest {
+
+   private final GetDocumentsDirectoryOperation instance
+           = new GetDocumentsDirectoryOperation();
+
+   /**
+    * Test of the validateParams method of class GetDirectoryOperation.
+    */
+   @Test
+   public void testValidateParams() {
+
+      String[] params;
+
+      // valid params - just the directory name
+      params = new String[]{"Directory"};
+      instance.validateParams(params);
+
+      // valid params - extra junk
+      params = new String[]{"Directory", "Other", "Other"};
+      instance.validateParams(params);
+
+      // invalid params - empty
+      params = new String[0];
+      try {
+         instance.validateParams(params);
+         fail("Expected IllegalArgumntException");
+         }
+      catch (IllegalArgumentException ex) {
+         }
```

Comment:
Actually, based on my original code, all arrays of lengths greater than one are valid as well. They're treated the same as empty arrays. Either implementation is fine, but you must validate empty arrays.